### DISCUSSION TITLE
Welcome to pylife Discussions!

### DISCUSSION THREAD

**Discussion by johannes-mueller** [boschresearch/pylife#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gregory-C** [comment]
Hi!
Thanks so much for open-sourcing this work! :-)

Regarding introducing myself, my name is Greg Campeau.  I like to predict how things could last longer.  But more importantly, as an engineer, I have learned to appreciate the efforts needed to adequately test these predictions.

Presently, I help with physical testing of various materials that many things are made of.

In the past, I helped research lubricant life, which helped me gain an appreciation for testing predictions.  

I have also helped with embedded controller testing.  From this, I saw that for testing to be useful, knowing what something is actually supposed to do is a necessity.

I also helped with some re-validation efforts for a couple of medical device companies.  From this, I saw that knowing what something is supposed to do must somehow be continuously-checked to catch evolving needs of users and others, who have a some sort of stake in the commerce or safety of the given something.

Regarding how I would like to help this community project, or at least how I think I would like to help, I am thinking that I would ultimately like to help with some aspect of the test-driven development efforts.   But in the near-term, while I reduce my ignorance of GitHub, Git, and Python, I imagine my help will be limited.  However, as I think about my prior community work, my most memorable contributions consisted of doing my best to be a non-judgmental person, as ideas and thoughts were bounced off of me.

**Comment by MaximLutovinov** [comment]
Greetings! 

My name is Maxim Lutovinov, I'm a FEA engineer at Akkodis in Prague. In my line of work I do strength and durability calculations for automotive industry. For fatigue calculations I'm using Femfat and Femsite. I've learnt about pyLife from my PhD study supervisor Jan Papuga and attended pyLife workshop at CTU in Prague last week. I was impressed by your programming culture - the code is neat, well commented and readable - and looking forward to exploring it more.
